### ISSUE TITLE
How to capture picture

### ISSUE THREAD

**Issue by eightHundreds** [mistic100/Photo-Sphere-Viewer#193]
I try capture picture by canvas toDataURL. But the result is always black.
Then i found that use THREE.CanvasRenderer will resolve this problem.
I try use  [`webgl: false,`](https://github.com/mistic100/Photo-Sphere-Viewer/blob/b5ad3aa005aaed3e181448ccda3c33fb0a8000d0/src/js/PhotoSphereViewer.defaults.js#L126) to change renderer.
Unfortunately, it still failed([`setPixelRatio`](https://github.com/mistic100/Photo-Sphere-Viewer/blob/b5ad3aa005aaed3e181448ccda3c33fb0a8000d0/src/js/PhotoSphereViewer.core.js#L425) is no a function)

**Comment by mistic100** [comment]
I guess you tried `PSV.renderer.domElement.toDataURL()`
See this issue https://github.com/mrdoob/three.js/issues/1472

I saw posts saying this has a negative impact on performances, so it won't be included in the lib.

**Comment by cyruscuenca** [comment]
I've tried running toDataURL on the canvas, and just get the markers and labels on a black background.

I looked into [mrdoob/three.js#1472](https://github.com/mrdoob/three.js/issues/1472) but I'm having trouble understanding the context for the issue. I don't really understand the Three.js instance, and where it's called.

I also can't seem to find where the panorama is being rendered in the DOM Inspector.

What do I need to Google to find more info on how to export the panorama in it's current position?
